src/main/query_profiler.cpp

```diff
@@ -721,18 +721,24 @@ void QueryProfiler::QueryTreeToStream(std::ostream &ss) const {
 	}
 }
 
-InsertionOrderPreservingMap<string> QueryProfiler::JSONSanitize(const InsertionOrderPreservingMap<string> &input) {
+Value QueryProfiler::JSONSanitize(const Value &input) {
```

Comment:
I wonder if we should somehow enforce that this has to be called with a `Value` or type `MAP`? Maybe an assertion would be good?